otj-server-core/src/main/java/com/opentable/server/ConservedHeadersJettyErrorHandler.java

```diff
@@ -47,7 +48,11 @@ public void setShowStacks(boolean showStacks) {
     public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException {
         if (this.delegate != null) {
             ConservedHeadersFilter.extractHeaders(request)
-                .forEach((header, value) -> response.setHeader(header.getHeaderName(), value));
+                .forEach((header, value) -> {
+                    if (HEADER_BLACKLIST.copyToResponse(header.getHeaderName())) {
+                        response.setHeader(header.getHeaderName(), value);
```

Comment:
I didn't even realize that we have logic in `otj-server` that also copies conserved headers to responses, apart from what already happens in `otj-conservedheaders`.

Would the be any way to consolidate this logic in one place?